icevision/models/torchvision_models/faster_rcnn/dataloaders.py

```diff
@@ -85,10 +85,14 @@ def _build_train_sample(
     if len(record["labels"]) == 0:
```

Comment:
Instead of modifying the faster_rcnn dataloader we should create a specific extended dataloader for keypoints_rcnn, similarly to what is done for mask_rcnn dataloaders

Reply:
Done!